Calculator/Layout.java

```diff
@@ -0,0 +1,63 @@
+import java.awt.Component;
+
+/**
+ * determine the layout
+ * @author Udit Saxena
+ *
+ */
+public class Layout {
+
+	private int screenSize[][];
```

Comment:
Screen size can be in just 2 variables: Height and width, No need to take complete 2D array